src/server/services/data-transform.service.js

```diff
@@ -106,11 +107,14 @@ const transformAnswersForSummary = (cumulativeFullAnswers, addressLookups) => {
         delete data.operator_postcode_find;
         delete data.operator_address_selected;
       }
+    } else {
+      data.operator_first_line = data.operator_address_line_1;
```

Comment:
Having looked at this again, I think we should also set operator_street to address line 2